pom.xml

```diff
@@ -23,7 +23,16 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
-
+    <dependency>
```

Comment:
nit: Is there a reason why this dependency was moved within a file? If not, please move it back.